src/App.js

```diff
@@ -49,8 +51,8 @@ class App extends Component {
                height={item.height}     // Height - used for a CSS style to position on the screen
                layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
                key={item.id}            // Key - to help React with performance
-
-               // Additional props (event callbacks, etc.) can be passed here
+               type={item.type}
+               itemClicked={this.itemClicked}
```

Comment:
Make sure that you are highlighting what the purpose of that function is with its name. If it’s a callback, use the term callback, if it’s an event handler, tell us what kind of event it is. 

Something like: `addPointCallback = {this.increaseScore}`. Save `itemClicked` for the event handler in GameItem.js